src/Example/Controllers/AccountController.cs

```diff
@@ -79,6 +86,20 @@ public ActionResult Connect(string token)
             }
         }
 
+        private static void UpdateAttributesIfPresent(YotiUserProfile yotiProfile, User user)
```

Comment:
Is this an attempt to align the host application user with the values from the profile using reflection?

Reply:
Since the .NET example is saving to a database, this method saves the values retrieved from the profile to an object which is then saved to the database